### ISSUE TITLE
[BUG/Feature/???] Input of negative fee not possible

### ISSUE THREAD

**Issue by dittodhole** [ghostfolio/ghostfolio#4022]
**Bug Description**

Bought IE00BM67HM91 some time ago, and paid some fee for accumulation. Now I've sold the mentioned ETF with loss, which results in a negative fee (costs for selling, minus the negative gains tax = absolute gain).

It is not possible anymore to input a negative fee.

See [Min-attribution](https://github.com/ghostfolio/ghostfolio/blob/316b7e82f1a9a8e03819e8339456826c7bfbe3e1/apps/api/src/app/order/create-order.dto.ts#L61), which got introduced with https://github.com/ghostfolio/ghostfolio/pull/2496

**To Reproduce**

Sell with negative fee.

**Expected behavior**

Negative fee should be possible, or another way of how to get correct final amount of transaction.

**Screenshots**

N/A

**Logs**

N/A

**Environment**

- Ghostfolio Version 2.121.1
- Self-hosted
- Experimental Features enabled
- Brave
- Arch

**Additional context**

N/A

**Comment by Bazeel-Waris** [comment]
@dittodhole  can I work it?

**Comment by dtslvr** [comment]
> can I work it?

Thanks for your interest, @Bazeel-Waris! Since you're already working on https://github.com/ghostfolio/ghostfolio/issues/3998, it would be great to finish that one first before picking up a new task.

**Comment by Bazeel-Waris** [comment]
Okay 👍

**Comment by LapplandsCohan** [comment]
Just adding to use cases:

When earning dividends on foreign shares, a tax (or fee in Ghostfolio as it doesn't differentiate between fees and taxes) that is too high is deducted from the amount received. This is corrected by a partial refund a while later (usually the next month, but sometimes not until the beginning of next year). For correct statistics this should be entered in Ghostfolio as a negative fee.

If I have dividends from several different foreign shares, the partial refund will be as a lump sum, so I can not associate the negative fee with a specific share.

**Comment by GilDev** [comment]
Still no news about this? I got some referral coupons refunded by my broker, in Portfolio Performance I can put a “Fees Refund” but I’m not sure how to do the same with Ghostfolio.
